docs/user-guide/services.md

```diff
@@ -388,23 +388,76 @@ as of Kubernetes 1.0.
 
 ### Type NodePort
 
-If you set the `type` field to `"NodePort"`, the Kubernetes master will
+If you set the `spec.type` field to `"NodePort"`, the Kubernetes master will
 allocate a port from a flag-configured range (default: 30000-32767), and each
 Node will proxy that port (the same port number on every Node) into your `Service`.
-That port will be reported in your `Service`'s `spec.ports[*].nodePort` field.
+Here is an example of a `Service` definition of type `"NodePort"` that can be issued using the
+[`kubectl run`] (https://cloud.google.com/container-engine/docs/kubectl/create) command.
 
-If you want a specific port number, you can specify a value in the `nodePort`
-field, and the system will allocate you that port or else the API transaction
-will fail (i.e. you need to take care about possible port collisions yourself).
-The value you specify must be in the configured range for node ports.
+```json
```

Comment:
yes it might seem to be simple, it just illustrates setting the specific port. The example above is based on the automatic port allocation. If you think it is a waste of 20 lines - I will pass. I think there more examples the documentation (even simple) the better for newcomers. IMHO .